msbuild/Xamarin.MacDev.Tasks.Core/VerbosityUtils.cs

```diff
@@ -91,20 +91,20 @@ static public string GetVerbosityLevel (string commandLine)
 
 		// The values here come from: https://github.com/mono/monodevelop/blob/143f9b6617123a0841a5cc5a2a4e13b309535792/main/src/core/MonoDevelop.Projects.Formats.MSBuild/MonoDevelop.Projects.MSBuild.Shared/RemoteBuildEngineMessages.cs#L186
 		// Assume 'Normal (2)' is the default verbosity (no change), and the other values follow from there.
-		static public string GetVerbosityLevel (LoggerVerbosity v)
+		static public string[] GetVerbosityLevel (LoggerVerbosity v)
 		{
 			switch ((LoggerVerbosity) v) {
 			case LoggerVerbosity.Quiet:
-				return "-q -q -q -q";
+				return new [] { "-q", "-q", "-q", "-q" };
 			case LoggerVerbosity.Minimal:
-				return "-q -q";
+				return new [] { "-q", "-q" };
 			case LoggerVerbosity.Normal:
 			default:
-				return String.Empty;
+				return Array.Empty<string> ();
 			case LoggerVerbosity.Detailed:
-				return "-v -v";
+				return new [] { "-v", "-v" };
 			case LoggerVerbosity.Diagnostic:
-				return "-v -v -v -v";
+				return new [] { "-v", "-v", "-v", "-v" };
```

Comment:
Why do we have repeated `-v -v` flags?

Reply:
How verbose we are is an int not a boolean:

https://github.com/xamarin/xamarin-macios/blob/master/tools/common/Driver.cs#L169

So slapping more -v gives you more details on your build.

Reply:
@prmallic it's like `msbuild`, you can get different level, but they are not named (just a counter like @chamons mentioned).

E.g. msbuild
```
 -verbosity:<level> Display this amount of information in the event log.
                     The available verbosity levels are: q[uiet], m[inimal],
                     n[ormal], d[etailed], and diag[nostic]. (Short form: -v)
                     Example:
                       -verbosity:quiet
```